src/Explorer/Controls/NotebookGallery/GalleryViewerComponent.tsx

```diff
@@ -222,8 +223,8 @@ export class GalleryViewerComponent extends React.Component<GalleryViewerCompone
       content: this.isEmptyData(data)
         ? this.createEmptyTabContent(
             "ContactHeart",
-            "You have not liked anything",
-            "Like any notebook from Official Samples or Public gallery"
```

Comment:
isnt this not ok? favorited looks very odd

Reply:
I tried to be consistent and use "Favorite" instead of "Like" everywhere. What do you suggest we should do? I think we can check this in and get everyone's opinion after using in MPAC?

Reply:
I'll note it down and have a PR with all string updates later on. Trying to check this in asap to get feedback.